CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+## 1.0.0 / 2025-02-XY
```

Comment:
I would not release this as 1.0. Just release as 0.4.0

Reply:
Okay. Do you think this release in particular should not be 1.0, or that this project should not release a 1.0 ever because of the underlying APIs not being stable, or?

Reply:
Maybe not "ever", but you're point about the underlying APIs not being stable is why most exporters don't do major releases.

Reply:
Okay, I’ll fix this up to reflect that.  And maybe I’ll recreate the PR with a less bad source branch name 🌞, when we get there. Though it looks like the source branch name isn’t included in the resulting `main` commit, so maybe that’s not necessary.

Regardless, this still needs #101 merged as well. Can you take a look at it?